### PR TITLE
Change eslint to airbnb

### PR DESCRIPTION
**Pull request by leighd2008**

### CODE REVIEW

File: frontend/src/notApp.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+import './App.css';
+import CommunityProjects from './Components/CommunityProjects/CommunityProjects';
+import CommunityPosts from './Components/CommunityPosts/CommunityPosts';
+import { projectList } from './projectList';
+import { posts } from './posts';
+
+class App extends Component {
```
Comment (Mitchell0625) [inline frontend/src/notApp.js]: Why the duplicate App component?

File: frontend/src/notApp.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    super();
+    this.state = {
+      route: 'home',
+      user: '',
+      projectList: projectList,
+      posts: posts,
+    }
+  }
+
+  onRouteChange = (route, index) => {
```
Comment (Mitchell0625) [inline frontend/src/notApp.js]: react-router-dom (React Router) allows for routes to be created in a cleaner fashion. It eliminates having to pass a route change function to each component.

https://reacttraining.com/react-router/core/api/Route

File: frontend/src/notApp.js (hunk 1 above)
Comment (leighd2008) [reply frontend/src/notApp.js]: it was an over site, should have been deleted.

File: frontend/src/notApp.js (hunk 2 above)
Comment (leighd2008) [reply frontend/src/notApp.js]: This file has been deleted

**Pull request merged**